docs/help/support-guidelines.md

```diff
@@ -1,5 +1,86 @@
 # Support guidelines
 
+This page explains the support mechanisms we provide.
+
 !!! warning
 
-    Under Construction
+    Before asking any help, search and read this documentation site carefully.
```

Comment:
```suggestion
    Before asking for help, please search and read this documentation site carefully.
```

Reply:
The word `please` is not recommended. So can you remove it?
https://developers.google.com/style/tone?hl=en#politeness-and-use-of-please

Reply:
Wow that is a great site, I will check the recommendations there before commenting here in the future, thanks!

Also updated.

Reply:
I will write documentation guidelines for Autoware in the near future!